From 0a358027b1f1172a1d379308836d9b4bf9683a98 Mon Sep 17 00:00:00 2001
From: Sailaja <sbandarupalli@habana.ai>
Date: Fri, 20 Jan 2023 09:55:03 +0200
Subject: [PATCH] [SW-118828] gaudi2: unify tpc0, 1 routing

When security is enabled re-routing of TPC0 and 1 is done to DCORE0_RTR1.
In other case its left with default values which routes to DCORE0_RTR0

As per LKD team, rerouting of tpc0 and tpc1 should be identical
regardless of security flag.

Change-Id: Ibba71a0263c829fda928fc1d47597006ae35ac87
Signed-off-by: Sailaja <sbandarupalli@habana.ai>
---

diff --git a/arch/gaudi2/common/security.c b/arch/gaudi2/common/security.c
index eda7331..1470a44 100644
--- a/arch/gaudi2/common/security.c
+++ b/arch/gaudi2/common/security.c
@@ -1288,10 +1288,10 @@
 {
 	preboot_secure_pcie();
 	preboot_security_sp_sram_init();
+	preboot_lbw_rtr_preconfig();
 
 	/* Do security related initializations only if security is enabled */
 	if (preboot_security_is_enabled()) {
-		preboot_lbw_rtr_preconfig();
 		hl_dev_prot_bits_init();
 		preboot_secure_pmmu();
 		preboot_secure_psoc();
